### PR TITLE
Renamed appearance tab to membrane tab.

### LINKED ISSUE #1869: Rename appearance tab to membrane tab
It was originally called membrane tab before the editor UI rework (and releases prior to 0.5.2), but I somehow renamed it to appearance. Membrane was actually agreed upon as the tab's name, see https://github.com/Revolutionary-Games/Thrive/pull/927#discussion_r365542717.

### PR DESCRIPTION
**Pull request by buckly90**
closes #1869

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
Need to run localization script before this can be merged.

**Comment by hhyyrylainen** [comment]
Also it seems that the localizations don't have plain MEMBRANE key in them, so a "translation" for MEMBRANE to "Membrane" needs to be added to en.po

File: locale/en.po [hunk 1]
```diff
@@ -1718,176 +1718,177 @@ msgstr "Auto-evo failed to run"
...
-#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1577
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1580
 msgid "STRUCTURE"
 msgstr "Structure"
 
-#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1600
-msgid "APPEARANCE"
-msgstr "Appearance"
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1603
+#, fuzzy
```
Comment (hhyyrylainen) [inline locale/en.po:1766]: This line should be removed because this is a proper translation, right?
Or should this translation actually be "Membrane"

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: locale/en.po (hunk 1 above)
Comment (hhyyrylainen) [reply locale/en.po:1766]: I just tested and it looks like Godot ignores translations marked as fuzzy entirely. So the tab actually now ends up being "MEMBRANE" which is not consistent with the text to its left or right.

**Review by FuJa0815 — COMMENTED** [review]
There are still merge conflicts. I can try to resolve them if you want me to.

**Comment by buckly90** [comment]
> 
> 
> There are still merge conflicts. I can try to resolve them if you want me to.

Sorry about the late reply, I had forgotten about this.
I would like to at least try to resolve the conflicts myself first as I don't like shirking work I started at the expense of others like that.

Any tips on how to go about this? The conflicting files just seem to be the files I changed, which I assume will overwrite any changes others have made in the event that my branch is outdated (It most likely is). My usual method of updating branches of mine is to just take out the files I changed, pull from master, and then replace them. But that's normally when I just change one or two.

**Comment by FuJa0815** [comment]
I would do the following:
1. `git pull`
2. `git merge master`
3. resolve the conflicts using any editor. I use Visual Studio Code

I resolve language files like that:
I go through one file manually and check if the current change is applicable everywhere. Line numbers don't really matter.
If that is the case I accept all current changes (VSC: Ctrl+Shift+P => Merge Conflict: Accept all Current) in all language files.
After resolving every conflict I rerun `update_localization.ru`. There will probably be some duplicate definitions you will have to resolve manually by removing one of them.

I resolve the scene file the following way:
Margin conflicts don't really matter. Choose whatever seems better.
Every other conflict requires case to case care.

After everything is done: `git commit -am "resolved merge conflicts with master" && git push`

File: src/microbe_stage/editor/MicrobeEditor.tscn [hunk 2]
```diff
@@ -1562,8 +1562,13 @@ custom_colors/font_color_pressed = Color( 0, 0, 0, 1 )
 toggle_mode = true
 action_mode = 0
 enabled_focus_mode = 0
+<<<<<<< HEAD
+group = SubResource( 21 )
+text = "MEMBRANE"
+=======
 group = SubResource( 15 )
 text = "APPEARANCE"
+>>>>>>> master
```
Comment (FuJa0815) [inline src/microbe_stage/editor/MicrobeEditor.tscn:1571]:
```suggestion
group = SubResource( 15 )
text = "MEMBRANE"
```

**Review by FuJa0815 — CHANGES_REQUESTED** [review]
There is a merge conflict present.
ID 15 should work, because the other tab buttons also use group 15 (see line 1542)

File: src/microbe_stage/editor/MicrobeEditor.tscn (hunk 2 above)
Comment (buckly90) [reply src/microbe_stage/editor/MicrobeEditor.tscn]: Strange, my editor wouldn't point that one out to me at all, so I had to find it manually.

Comment (FuJa0815) [reply src/microbe_stage/editor/MicrobeEditor.tscn]: It isn't fixed, there is still one bad line (`>>>>>>> master`). Fix the file via a text editor.

Comment (buckly90) [reply src/microbe_stage/editor/MicrobeEditor.tscn]: Should be gone now, apologies for the time.

**Comment by hhyyrylainen** [comment]
Looks like this only has part of the required changes, that isn't consistent between the translation files.

**Review by Kasterisk — APPROVED** [review]
Looks to be correct codewise and ingame, I'd say this is now good to merge 👍

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**